eos/form-factors/parametric-bgjvd2019-impl.hh

```diff
@@ -185,6 +186,13 @@ namespace eos
         _m_B(p[Process_::name_B], *static_cast<ParameterUser *>(this)),
         _m_P(p[Process_::name_P], *static_cast<ParameterUser *>(this))
     {
+        static const Log::OneTimeMessage message_HQET_FFs_PToP
+        (
+            std::string("HQETFormFactors<") + Process_::name_B + "->" + Process_::name_P + ",PToP>",
```

Comment:
I just tested this in a notebook. I find that both the B->P and B->V message are displayed. And they are display only once, too!

Small issue: ``Process_::name_B`` refers to the parameter name for the B-meson mass. My bad. Can you replace
this with
```c++
"HQETFormFactors<" + Process_::label + ",PToP>"
```
instead? Of course, in all four occurrences!

Reply:
ok, I will implement your suggestions
In the command line it displays only PToP warning, but it is not a big issue